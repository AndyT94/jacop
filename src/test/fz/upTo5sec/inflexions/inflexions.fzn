array [1..8] of var int: x:: output_array([1..8]) = [1,1,4,8,8,2,7,1];
solve :: int_search(x,first_fail,indomain_min,complete) satisfy;
