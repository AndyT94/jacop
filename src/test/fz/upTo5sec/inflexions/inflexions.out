x = array1d(1..8, [1, 1, 4, 8, 8, 2, 7, 1]);
----------